index.html

```diff
@@ -0,0 +1,131 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <!-- Required meta tags -->
+    <meta charset="utf-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, shrink-to-fit=no"
+    />
+
+    <!-- Bootstrap CSS -->
+    <link
+      rel="stylesheet"
+      href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css"
+      integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh"
+      crossorigin="anonymous"
+    />
+
+    <title>Library</title>
+  </head>
+  <body>
+    <h1 class="display-4 text-center py-2">
+      My <span class="text-primary">Book</span> List
+    </h1>
+    <hr />
+    <div class="container">
+      <div class="row">
+        <div class="col-lg-4 col-md-4 col-12">
+          <div class="mx-auto pt-2">
+            <div id="message"></div>
+            <form id="form-book">
+              <div class="form-group">
+                <label for="title">Title</label>
+                <input
+                  type="text"
+                  class="form-control"
+                  id="title"
+                  placeholder="Book Title"
+                />
+              </div>
+              <div class="form-group">
+                <label for="name">Author</label>
+                <input
+                  type="text"
+                  class="form-control"
+                  id="author"
+                  placeholder="Author Name"
+                />
+              </div>
+              <div class="form-group">
+                <label for="name">Pages</label>
+                <input
+                  type="text"
+                  class="form-control"
+                  id="pages"
+                  placeholder="Page Number"
+                />
+              </div>
+              <div class="form-check">
+                <input
+                  class="form-check-input"
+                  type="radio"
+                  name="status"
+                  id="read"
+                  value="Read"
+                  checked
+                />
+                <label class="form-check-label" for="read">
+                  Read the Book
+                </label>
+              </div>
+              <div class="form-check">
+                <input
+                  class="form-check-input"
+                  type="radio"
+                  name="status"
+                  id="unread"
+                  value="Unread"
+                />
+                <label class="form-check-label" for="unread">
+                  Not Read the book
+                </label>
+              </div>
+              <button type="submit" class="btn btn-primary mt-2">
+                Add Book
+              </button>
```

Comment:
- This button should have the functionality [described here](https://www.theodinproject.com/courses/javascript/lessons/library)
![image](https://user-images.githubusercontent.com/46115725/72383240-a9595680-36e0-11ea-91a1-507cb709ef74.png)
 The button is not bringing up a form rather than just taking care of the submission, which should be inside of the form that the above mention button is referring to.